cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -187,6 +206,121 @@ void test_outlet_rename() {
         ;
     }
 
+    @Test
+    void test_rating_spec_id() {
+        // Structure of test:
+        // 1) Create the Rating Location group
```

Comment:
I'd expect this organized like this:

1. Create the outlet
2. Retrieve the outlet, and verify that the rating-spec-id is null
3. Retrieve the location group using the outlet's category and group id, deserialize the body to an object
4. Modify the location group shared-loc-alias-id, then store it back to the database
5. Retrieve your outlet, and verify the rating-spec-id comes back as expected
6. Delete the outlet
7. Delete the location group

This verifies that:
- The automatic DB location group creation for outlets is working as expected, and starts off with null
- When you change the location group's shared-loc-alias-id it's reflected in the outlet

We don't need to verify that the outlet was deleted, that's already part of the crud testing.
Don't validate the behavior of the location groups API, it's already tested in other integration tests.

Reply:
Updated test case to match this pattern